### DISCUSSION TITLE
Traffic Control

### DISCUSSION THREAD

**Discussion by Dtyhimlad** [WGDashboard/WGDashboard#475] [Ideas]
Just a thought, I was implementing this in my server and thought it would be useful out of the box.
Leaving this here for reference
https://gist.github.com/danpawlik/078552bb3b2ee2f38b04707ab539a178

**Comment by DaanSelen** [comment]
Hi there! Can you explain more about what this is and what for? I am not known with Openwrt

**Comment by Dtyhimlad** [comment]
Its for controling traffic speeds for peers,
given for example a network speed of 500mb/sec you can limit Peer speed to guarantee given speed.

Enviado de Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Dtyhimlad** [comment]
There is some good documentation for it here
https://www.procustodibus.com/blog/2022/12/limit-wireguard-bandwidth/
Its class based, you limit by raio where 1:1 is full speed as an example.

**Comment by DaanSelen** [comment]
Thanks! Looks good, if you want to append it to the documentation somewhere you can submit something to that repo! https://github.com/donaldzou/WGDashboard-Documentation/tree/main/Writerside/topics

Or use it from here :)

**Comment by donaldzou** [comment]
Let me do some homework lol
